src/core/components/auth/auth-item-access-details.jsx

```diff
@@ -0,0 +1,63 @@
+import React from "react"
+import PropTypes from "prop-types"
+import ImPropTypes from "react-immutable-proptypes"
+
+export default class AuthsAccessDetails extends React.Component {
```

Comment:
Please add documentation explaining what the class does. Is there a reason the class name doesn't match the file name (I would expect camel case to hyphen conversion without drop in words).

Reply:
The file name changed as per suggestion.